testing/infra/terraform/modules/benchmark_executor/instance.tf

```
@@ -30,18 +42,17 @@ module "vpc" {
     }
   ]
 
-  tags = {
-    Owner       = var.user_name
-    Environment = "dev"
-  }
+  tags = merge(local.ci_tags, var.tags)
+
   vpc_tags = {
-    Name = "vpc-${var.user_name}-worker"
+    name = "vpc-${var.user_name}-worker"
```

`Name` is a special tag that is used for resource naming, it should have the Title format.

```suggestion
    Name = "vpc-${var.user_name}-worker"
```